state-processor/src/main/java/com/ververica/flink/training/exercises/ReadRidesAndFaresSnapshot.java

```diff
@@ -0,0 +1,66 @@
+package com.ververica.flink.training.exercises;
```

Comment:
Is this actually an exercise, or an example? Examples should be in `com.ververica.flink.training.examples`, I guess

Reply:
As we discussed, I think this is an exercise that doesn't involve any coding.